opentok.gemspec

```diff
@@ -28,5 +28,5 @@ Gem::Specification.new do |spec|
   spec.add_dependency "addressable", "~> 2.3" #  2.3.0 <= version < 3.0.0
   spec.add_dependency "httparty", "~> 0.15.5"
   spec.add_dependency "activesupport", ">= 2.0"
-  spec.add_dependency "jwt", "~> 1.5.6"
+  spec.add_dependency "jwt", "~> 2.1"
```

Comment:
Can we change this to `>= 1.5.6` ? This way it won't be a breaking change and folks would be able to upgrade to whichever version?